CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Smithy Changelog
 
+## 1.13.1 (2021-11-02)
+
+### Bug Fixes
+
+* Fixed a bug that the `apply` transform to not run its projections. ([#969](https://github.com/awslabs/smithy/pull/969))
```

Comment:
think you're missing "caused" in there (caused the apply transform...)

Reply:
Yes...